berkeley-mobile/Calendar/Campus/CampusEventDetailViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  CampusEventDetailViewController.swift
+//  berkeley-mobile
+//
+//  Created by Shawn Huang on 9/19/20.
+//  Copyright © 2020 ASUC OCTO. All rights reserved.
+//
+
+import UIKit
+
+fileprivate let kViewMargin: CGFloat = 16
+
+class CampusEventDetailViewController: UIViewController {
+    var barView: BarView!
+    var event: EventCalendarEntry!
+    var overviewCard: EventOverviewCardView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        setUpBackgroundView()
+        setUpScrollView()
+        setUpOverviewCard()
+        setupDescriptionCard()
+        setUpLearnMoreButton()
+        setUpRegisterButton()
+    }
+    
+    func setUpBackgroundView() {
+        view.backgroundColor = Color.modalBackground
+        view.layer.cornerRadius = 15
+        view.layer.maskedCorners = [.layerMaxXMinYCorner, .layerMinXMinYCorner]
+        view.clipsToBounds = true
+        
+        barView = BarView(superViewWidth: view.frame.width)
+        view.addSubview(barView)
+    }
+    
+    func setUpScrollView() {
+        view.addSubview(scrollingStackView)
+        scrollingStackView.topAnchor.constraint(equalTo: barView.bottomAnchor, constant: kViewMargin).isActive = true
+        scrollingStackView.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor).isActive = true
+        scrollingStackView.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor).isActive = true
+        scrollingStackView.bottomAnchor.constraint(equalTo: view.layoutMarginsGuide.bottomAnchor).isActive = true
+    }
+    
+    func setUpOverviewCard() {
+        overviewCard = EventOverviewCardView(event: event)
+        overviewCard.heightAnchor.constraint(equalToConstant: 174).isActive = true
```

Comment:
Removing this constraint causes the overview card to grow in height dynamically depending on the event name. What do you think about this behavior? Would the inconsistency in size for this card for different events would be jarring?

Reply:
instead of removing it I set a height anchor <= 200 so that it doesn't expand infinitely/too much even if backend sends some messed up/too long data. looks good with the resize so there's less empty space. almost all if not all names should fit.

Reply:
![IMG_1791](https://user-images.githubusercontent.com/20230668/97118801-6e516e00-16c9-11eb-9aa1-100fc2ade6b7.PNG)
![IMG_1790](https://user-images.githubusercontent.com/20230668/97118805-6f829b00-16c9-11eb-9afe-29a6c4932dd3.PNG)